### PR TITLE
fix: link colors and open external in dynamic airtable HTML #983

### PR DESCRIPTION
**Pull request by andregoncalves**
Fixes issues with anchor tags in the dynamically loaded html from airtable.
- Color of links
- Links open in external browser window

### CODE REVIEW

File: src/renderer/marketplace/incorporation/detail/index.jsx [hunk 1]
```diff
@@ -155,6 +155,9 @@ const styles = theme => ({
 		'& ul li': {
 			lineHeight: '1.4em',
 			marginBottom: '0.5em'
+		},
+		'& a': {
+			color: '#93B0C1'
```
Comment (sk91) [inline src/renderer/marketplace/incorporation/detail/index.jsx]: @andregoncalves maybe better: 
`color: theme.palette.secondary.main`
or `import {typography} from 'selfkey-ui"`
`color: typography`

https://github.com/SelfKeyFoundation/selfkey-ui/blob/master/src/colors.tsx#L3
https://github.com/SelfKeyFoundation/selfkey-ui/blob/master/src/theme/selfkey-dark-theme.tsx#L48

**Comment by rodrigopavezi** [comment]
@andregoncalves would it make your life easier when our electron app will open all target="_blank" links externally. Because I am adding the bellow for that.

```
mainWindow.webContents.on('new-window', function(e, url) {
  e.preventDefault();
  require('electron').shell.openExternal(url);
});
```
Cheers

**Comment by andregoncalves** [comment]
@rodrigopavezi that definitely makes it easier. Is this is going into the release right now? Should I remove the click capture and handling?

**Comment by rodrigopavezi** [comment]
@andregoncalves oh, do you need it for this release? I thought otherwise because you are merging to dev branch. we can add it to this release for sure and you can remove the handling here. Cheers

File: src/renderer/marketplace/incorporation/detail/index.jsx (hunk 1 above)
Comment (andregoncalves) [reply src/renderer/marketplace/incorporation/detail/index.jsx]: Got it.

**Pull request merged**